tests/dummy/config/environment.js

```diff
@@ -18,6 +18,10 @@ module.exports = function(environment) {
       // when it is created
     },
 
+    'ember-metrics': {
```

Comment:
Since segment and google-tag-manager are not references in `metricsAdapters` but used in the unit tests.. they're excluded by default.  This proves the feature works :)

I can write additional tests by creating a dummy adapter in the test dummy and checking if its present still.